docs/getting_started/creating_mappings.md

```diff
@@ -0,0 +1,119 @@
+# Creating mappings
+
+With your chemical models loaded and defined into two ``ChemicalSystem`` objects,
```

Comment:
I'd make the entry point for mappings just knowing how to build a SmallMoleculeComponent? (i.e. you're not using the ChemicalSystem as an input to mapping generation). Part of the rationale for limiting the scope of the input is that someone might use openfe purely for mappings and then just doesn't need to know anything about ChemicalSystem.